qcore/config.py

```diff
@@ -62,3 +62,4 @@ class ConfigKeys(Enum):
 
 host, host_config_path = determine_machine_config()
 qconfig = get_machine_config(config_path=host_config_path)
+module_requirments = join(qconfig["tools_dir"], "module_requirements")
```

Comment:
Can we get a copy of module requirements in the repo or on the wiki?

Reply:
https://wiki.canterbury.ac.nz/display/QuakeCore/Loading+Modules+for+specific+tools_dir+with+different+compilers